.travis.yml

```diff
@@ -36,9 +38,9 @@ script:
 branches:
   only:
   - master
-  - /^(dawn)\/.*$/
-  - /^(runtime)\/.*$/
-  - /^(test)\/.*$/
+  - /^dawn\/.*$/
+  - /^runtime\/.*$/
+  - /^test\/.*$/
 deploy:
```

Comment:
The glob on line 48 no longer matches, so this will actually break builds. You can test by tagging a copy of your PR code as `test/dawn/...` or `test/runtime/...` and verifying that artifacts are actually pushed to github.